docs/features/software-catalog/descriptor-format.md

```diff
@@ -1001,6 +1010,15 @@ belongs to, e.g. `artist-engagement-portal`. This field is optional.
 | --------------------------------------- | ------------------------------------------ | ----------------------------------------------------------------------------- |
 | [`System`](#kind-system) (default)      | Same as this entity, typically `default`   | [`partOf`, and reverse `hasPart`](well-known-relations.md#partof-and-haspart) |
 
+### `spec.dependencyOf` [optional]
+
+An array of [entity references](#string-references) to the Components that the
+resource is a dependency of, e.g. `artist-lookup`. This field is optional.
+
+| [`kind`](#apiversion-and-kind-required)  | Default [`namespace`](#namespace-optional) | Generated [relation](well-known-relations.md) type                                            |
+| ---------------------------------------- | ------------------------------------------ | --------------------------------------------------------------------------------------------- |
+| [`Component`](#kind-component) (default) | Same as this entity, typically `default`   | [`dependencyOf`, and reverse `dependsOn`](well-known-relations.md#dependson-and-dependencyof) |
```

Comment:
Open question here too, should we really open up for the reverse dependency in the spec? It feels like a pretty fringe thing.

Reply:
We could argue that it we do it for groups and users too, but on first sign I would say that my component references something. I think we should be flexible.

Reply:
Ok so vote to keep this, also with no default?

But the tables (here and above) should list both Resource and Component as alternatives I think

Reply:
I think in this case, from the Resource direction outward, the only dependencyOf relation that might make sense would be a `Component` though, right?

Reply:
Not sure! This is one of the reasons this wasn't in place yet I guess, it's a little hazy still what the details should be :)

Again, probably we'll have to not restrict, and leave without a default.

Reply:
I'm still that adding `Resource` here wouldn't be semantically appropriate, as it would allow the declaration of `Resource` dependencyOf `Resource`.  I don't know if that's useful or not?

I'm tending to agree with you that this side of the declaration of the relationship might not be all that useful.  Much more reasonable to say that my `Component` dependsOn `Resource` which was the original intent of the Issue.